docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,165 @@
+.. _acsincome-data:
+Retiring Adult: ACSIncome
+-------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The ACSIncome dataset is one of five datasets created by Ding et al. [#0]_ 
+as an improved alternative to the popular UCI Adult dataset.
+The authors compiled data from the American Community Survey (ACS) Public Use Microdata Sample (PUMS). 
+Note that this is a different source than the Annual Social and Economic Supplement (ASEC) 
+of the Current Population Survey (CPS) used to construct the original UCI Adult dataset.
+Ding et al. [#0]_ filtered the data such that ACSIncome only includes individuals above 16 years old 
+who worked at least 1 hour per week in the past year and had an income of at least $100.
+
+
+
+
+.. _acsincome_dataset_description:
+
+Dataset Description
+^^^^^^^^^^^^^^^^^^^
+Data is provided from 2018 for all 50 states and Puerto Rico.
+There are 1,664,500 rows and 10 features, which are described below:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 7 30
+   :stub-columns: 1
```

Comment:
It's a bit hard to imagine what this looks like when built. Can you run the doc build command and paste what this looks like locally for you?

Reply:
Admittedly, I just copied the table settings from the boston housing file. This is how it looks on my end. Let me know if I should make changes
<img width="800" alt="Screen Shot 2022-01-11 at 4 05 33 PM" src="https://user-images.githubusercontent.com/31116084/149021373-1eecb9ed-2ab8-45ee-b379-375169a69477.png">

Reply:
Looks fine! Thank you